public/stylesheets/admin.css

```diff
@@ -90,3 +90,44 @@
 #legend-table>tbody>tr>td {
     padding: 2px 8px;
 }
+
+.mapbox-logo {
```

Comment:
Was this added in accidentally, or was it added for a reason?

Reply:
Accidental, no idea how that got there, thanks for the catch!!